script2.js

```diff
@@ -0,0 +1,12 @@
+// --------------------- Mobile-Menu------------------
+
+const mobileMenu = document.querySelector('.main-menu');
+const closeBtn = document.getElementById('close');
+const hamburger = document.getElementById('menu-bar');
+
+hamburger.onclick = function showMenu() {
+  mobileMenu.style.display = 'block';
+};
+closeBtn.onclick = function closeMenu() {
+  mobileMenu.style.display = none = 'none'; // eslint-disable-line
```

Comment:
- [ ] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: